#service for gettin control_signals from controller
std_msgs/Empty empty
---
last_letter_2/Control_signals signals